src/tests/class_tests/openms/source/BasicProteinInferenceAlgorithm_test.cpp

```diff
@@ -140,4 +140,43 @@ START_TEST(BasicProteinInferenceAlgorithm, "$Id$")
     }
     END_SECTION
 
+    START_SECTION(BasicProteinInferenceAlgorithm on Protein Peptide ID with grouping plus resolution and user set score type)
+    {
+      vector<ProteinIdentification> prots;
+      vector<PeptideIdentification> peps;
+      IdXMLFile idf;
+      idf.load(OPENMS_GET_TEST_DATA_PATH("newMergerTest_out.idXML"),prots,peps);
+      BasicProteinInferenceAlgorithm bpia;
+      Param p = bpia.getParameters();
+      p.setValue("min_peptides_per_protein", 0);
+      p.setValue("annotate_indistinguishable_groups", "true");
+      p.setValue("greedy_group_resolution", "true");
+      p.setValue("score_type", "RAW");  // should use the XTandem score meta value
+      bpia.setParameters(p);
+
+      TEST_EQUAL(peps[0].getScoreType(), "Posterior Error Probability"); // check if main score is PEP
+      bpia.run(peps, prots);
+      TEST_EQUAL(peps[0].getScoreType(), "Posterior Error Probability"); // check if main score has been reset again to PEP
+
+      TEST_EQUAL(prots[0].getHits().size(), 5)
```

Comment:
this is one more than before???
is this expected when switching to the RAW score?